### ISSUE TITLE
Cutter_Arm collides with Servo_Mount

### ISSUE THREAD

**Issue by 24Tigger24** [kevinakasam/ERCF_Filament_Cutter#8]
Hi, the Cutter_Arm collides with the Servo_Mount in the V2.0  
![tempsnip](https://github.com/kevinakasam/ERCF_Filament_Cutter/assets/155834359/03018072-8ff8-4f87-837a-6a49401a22e9)

**Comment by BioCookieYT** [comment]
Hi!
Sorry for the late answer and thanks for reporting this, I will work on a solution ASAP. It's a bit difficult time wise at the moment.

**Comment by BioCookieYT** [comment]
Please try the new Beta, it should be fixed now. Still have to update the Readme a bit and give some credits :p

**Comment by iamnotahippy** [comment]
I have printed the beta6 stls, and while I don't have my cutter wired up yet, the cutter arm no longer collides with the mount, and the gears no longer climb up one another when they encounter the resistance of cutting the filament. Also, I appreciate tapering the hole of the filament path through the cutter arm. I had to make the same modification to mine earlier to prevent jamming. Excellent work all the way around!
